projects/wave-core-new/src/lib/operators/dialogs/point-in-polygon-filter/point-in-polygon-filter.component.ts

```diff
@@ -65,9 +65,25 @@ export class PointInPolygonFilterOperatorComponent {
                                         workflowId,
                                         name,
                                         symbology: PointSymbology.fromPointSymbologyDict({
-                                            radius: {static: 10},
-                                            stroke: {width: {static: 1}, color: {static: [0, 0, 0, 255]}},
-                                            fillColor: {static: colorToDict(this.randomColorService.getRandomColorRgba())},
+                                            type: 'point',
+                                            radius: {
+                                                type: 'static',
+                                                value: 10,
+                                            },
+                                            stroke: {
+                                                width: {
+                                                    type: 'static',
+                                                    value: 1,
+                                                },
+                                                color: {
+                                                    type: 'static',
+                                                    color: [0, 0, 0, 255],
+                                                },
```

Comment:
We should really think about having the type for static stuff always here. I guess it's good for programming logic but looks very weird.

Reply:
it's not pretty but its constistent to always have the type